src/main/webapp/resources/js/pages/user/components/UserTable.jsx

```diff
@@ -56,7 +56,7 @@ export function UserTable() {
       fixed: "left",
       render(text, full) {
         return IS_ADMIN ? (
-          <a className="t-username" href={setBaseUrl(`users/${full.id}`)}>
+          <a className="t-username" href={setBaseUrl(`admin/users/${full.id}`)}>
```

Comment:
Move this `setBaseUrl` out of the columns declaration so it does not get called on each re-render.  Just set it once outside the component (just above the declaration).  All you need here is `${ BASE_URL}/${full.id}`

Reply:
Please see 76696f26e0d14c5aec0ae664becfb4d8cf83643a.